File: VB/DXWebApplication23/Models/DataHelper.vb
Imports System
Imports System.Linq
Imports DevExpress.Spreadsheet

Namespace DXWebApplication23.Models
	Public Class DataHelper
		Public Shared Function GetDocument() As Byte()
			Dim context As New DocumentsEntities()
			Return context.Docs.FirstOrDefault().DocBytes.ToArray()
		End Function

		Public Shared Sub SaveDocument(ByVal bytes() As Byte)
			Dim context As New DocumentsEntities()
			context.Docs.FirstOrDefault().DocBytes = bytes
			context.SaveChanges()
		End Sub
	End Class

	Public Class SpreadsheetData
		Public Property DocumentId() As String
		Public Property DocumentFormat() As DocumentFormat
		Public Property Document() As Byte()
	End Class
End Namespace

File: CS/DXWebApplication23/Models/DataHelper.cs
using System;
using System.Linq;
using DevExpress.Spreadsheet;

namespace DXWebApplication23.Models {
    public class DataHelper {
        public static byte[] GetDocument() {
            DocumentsEntities context = new DocumentsEntities();
            return context.Docs.FirstOrDefault().DocBytes.ToArray();
        }

        public static void SaveDocument(byte[] bytes) {
            DocumentsEntities context = new DocumentsEntities();
            context.Docs.FirstOrDefault().DocBytes = bytes;
            context.SaveChanges();
        }
    }

    public class SpreadsheetData {
        public string DocumentId { get; set; }
        public DocumentFormat DocumentFormat { get; set; }
        public byte[] Document { get; set; }
    }
}